ui/package.json

```diff
@@ -29,7 +29,7 @@
     "eslint-config-prettier": "^9.1.0",
     "ethers": "^5.7.1",
     "jest": "^28.1.3",
-    "next": "14",
```

Comment:
@amochuko Is it necessary to downgrade Next.js?

Reply:
No it's not. There isn't any need doing so.